interactions/api/models/member.pyi

```diff
@@ -16,7 +16,7 @@ class Member(DictSerializerMixin):
     user: Optional[User]
     nick: Optional[str]
     avatar: Optional[str]
-    roles: List[Role]
+    roles: Union[List[Role], List[int]]
```

Comment:
```suggestion
    roles: List[int]
```

Upon close investigation, we've type hinted this incorrectly the whole time. The API only returns a list of integers, and there's no way to request a specific role via. ID from the endpoints without us checking from a list of role objects via. `GET Guild Roles`.